src/Microsoft.DotNet.Cli.Utils/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System.IO;
+using Microsoft.DotNet.ProjectModel;
+using NuGet.Frameworks;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal static class ProjectContextExtensions
+    {
+        public static string GetOutputPath(this ProjectContext context, string configuration, string currentOutputPath)
+        {
+            var outputPath = string.Empty;
+
+            if (string.IsNullOrEmpty(currentOutputPath))
+            {
+                outputPath = Path.Combine(
+                    GetDefaultRootOutputPath(context, currentOutputPath),
```

Comment:
I know this is moved code and it was like this, so no need to fix in this commit.
Just want to call out funny behavior where this expression always returns the empty string, so technically it can be removed.